docs/modules/ROOT/pages/explanation/client-throttling.adoc

```diff
@@ -0,0 +1,22 @@
+= Client Throttling
+
+You may see the following similar error message from time to time when interacting with {product} on the command line, especially when using `kubectl` instead of `oc`:
```

Comment:
```suggestion
You may see a similar error message to the one below when interacting with {product} on the command line, especially when using `kubectl` instead of `oc`:
```